portality/settings.py

```diff
@@ -1402,6 +1402,11 @@
     'long_running': 93600,  # 26 hours
 }
 
+BG_MONITOR_DEFAULT_CONFIG = {
```

Comment:
Just add comment to the effect of 'Default monitoring config for background job types not enumerated in BG_MONITOR_ERROS_CONFIG below'

Reply:
added comment